projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/deploy/JobImageBuilder.java

```diff
@@ -176,6 +179,7 @@ public boolean buildImage(
         builderJobName,
         dockerRegistryService.builderImage(),
         false,
+        readOnlyRootFilesystem,
```

Comment:
JobImageBuilder is responsible for building docker images for data jobs. The builder job shouldn't be affected by this PR.

Reply:
Just defaulting to false?

Reply:
It does default to false. What should it default to?

Reply:
Yes, but if dataJob.readOnlyRootFileSystem is passed (value: true), it will affect the builder job, which is incorrect. The dataJob.readOnlyRootFileSystem property should affect only the data job.

Reply:
gotcha

Reply:
amended to have readOnlyRootFilesystem set to false always for builder jobs

Reply:
Thanks